modules/living_spaces_group/living_spaces_group.module

```diff
@@ -68,6 +68,18 @@ function living_spaces_group_menu_links_discovered_alter(&$links) {
   }
 }
 
+/**
+ * Implements hook_entity_presave().
+ */
+function living_spaces_group_entity_presave(EntityInterface $entity) {
+  /** @var \Drupal\Core\Entity\ContentEntityTypeInterface $definition */
+  $definition = \Drupal::entityTypeManager()->getDefinition($entity->getEntityTypeId());
+
+  if ($definition->getRevisionTable()) {
```

Comment:
fyi tried with `if (method_exists($entity, 'setNewRevision') && $entity instanceof RevisionableInterface) {}` but anyway it throws an error for the user entity type.
so, decided to use the check by revision table that entity types provide in annotation :)